### PR TITLE
chore: Adjust post-build script to add license files for sourcemaps on build

### PR DESCRIPTION
**Pull request by susnux**
## Summary

sourcemap and js share the same license, so we can simply create a copy from the license files,
or like here to save some space use symlinks.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by AndyScherzinger — APPROVED** [review]
Nice work @susnux 🎉

File: build/npm-post-build.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# Build CSS files from SCSS
+npm run sass
+# Build icons
+npm run sass:icons
+
+# Add licenses for source maps
+if [ -d "dist" ]; then
+	for f in dist/*.js; do
+		# If license file and source map exists copy license for the sourcemap
```
Comment (nickvergessen) [inline build/npm-post-build.sh:16]:
```suggestion
		# If license file and source map exists copy license for the source map
```

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**